driver-core/src/main/java/com/datastax/driver/core/policies/LatencyAwarePolicy.java

```diff
@@ -367,20 +371,45 @@ public long getMeasurementsCount() {
         }
     }
 
+    /**
+     * A set of DriverException subclasses that we should prevent from updating the host's score.
+     * The intent behind it is to filter out "fast" errors: when a host replies with such errors,
+     * it usually does so very quickly, because it did not involve any actual
+     * coordination work. Such errors are not good indicators of the host's responsiveness,
+     * and tend to make the host's score look better than it actually is.
+     */
+    private static final Set<Class<? extends DriverException>> EXCLUDED_EXCEPTIONS = ImmutableSet.of(
```

Comment:
These are the errors that we want to exclude when updating the host's score. There has been some hesitation about which errors to exclude, specially QueryValidationExceptions. I decided to include those too; please feel free to disagree.